misk-gcp/build.gradle

```diff
@@ -14,3 +14,10 @@ dependencies {
   testCompile project(':misk-testing')
   testCompile project(':misk-gcp-testing')
 }
+
+afterEvaluate { project ->
```

Comment:
can we avoid having to set this in every sub project?

Reply:
I tried numerous ways to move it to the root `build.gradle` without success, though I agree that'd be best. OkHttp and Barber also seem to have the same stanza copy-pasta in all of the subprojects.